drivers/console/Kconfig

```diff
@@ -210,6 +210,33 @@ config UART_PIPE_ON_DEV_NAME
 	  This option specifies the name of UART device to be used
 	  for pipe UART.
 
+config UPIPE_DEV_NAME_0
+	string "Device Name of UART Device 0 for pipe UART"
+	default "$(dt_chosen_label,$(DT_CHOSEN_Z_UART_PIPE))" if HAS_DTS
+	default "UART_2"
+	depends on UART_PIPE
```

Comment:
Does this actually depend on UART_PIPE anymore? bm_serial is just directly using the UART driver at this point, so I think all mentions and dependencies to upipe/uart_pipe can be dropped.